### DISCUSSION TITLE
How to "Install" Nightly

### DISCUSSION THREAD

**Discussion by skeelkat** [rustdesk/rustdesk#4027] [Q&A]
Can anyone help me on the correct procedure to install the nightly exe package? as far as I can tell it simply runs the nightly version but the older official version, is the one that still runs as a service in the background.

Thanks

**Comment by WhiteAls** [comment]
> Can anyone help me on the correct procedure to install the nightly exe package? as far as I can tell it simply runs the nightly version but the older official version, is the one that still runs as a service in the background.
> 
> Thanks

you need to run the nightly with `--install` flag
